static/js/ducks/allocations.js

```diff
@@ -9,9 +9,12 @@ const FETCH_ALLOCATIONS_OK = "skyportal/FETCH_ALLOCATIONS_OK";
 const REFRESH_ALLOCATIONS = "skyportal/REFRESH_ALLOCATIONS";
 
 // eslint-disable-next-line import/prefer-default-export
-export function fetchAllocations() {
-  return API.GET("/api/allocation", FETCH_ALLOCATIONS);
-}
+export const fetchAllocations = (params = {}) => {
+  if (!Object.keys(params).includes("apiType")) {
+    params.apiType = "api_classname";
+  }
```

Comment:
I wonder if we need this default since you already specify it explicitly in the components that use this fetch, and I am guessing the API can handle calls without this parameter...

Reply:
Yes good point.